src/condor_scripts/get_htcondor

```diff
@@ -139,6 +139,30 @@ check_forked() {
 
 
 get_distribution() {
+	if [[ $DISTRIBUTION ]]; then
+		#
+		# For testing purposes.  DISTRIBUTION is a space-separated tuple:
+		#      ${lsb_dist} ${dist_version} ${dist_version_number}
+		# where $dist_version number is only set on Debian.  Because the list
+		# is space-separated, it must be quoted (or escaped) on the command-
+		# line.  The values generated by this script for the supported
+		# platforms follow.
+		#
+		#     debian stretch 9
+		#     debian buster 10
+		#     ubuntu bionic
+		#     ubuntu focal
+		#     centos 7
+		#     centos 8
+		#     amzn 2
+		#
+
+		lsb_dist=$(echo $DISTRIBUTION | cut -d' ' -f 1)
+		dist_version=$(echo $DISTRIBUTION | cut -d' ' -f 2)
+		dist_version_number=$(echo $DISTRIBUTION | cut -d' ' -f 3)
```

Comment:
Looks like this will work fine if you have a single space as a separator.  If you want to allow arbitrary whitespace you can use `awk` instead.

Or, if you like, you could use the `read` builtin:

```suggestion
		read lsb_dist dist_version dist_version_number <<< $DISTRIBUTION
```